graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -39,23 +39,27 @@ def run_query(self, query: str, params: Dict[str, Any] = {}) -> DataFrame:
         if "gds.graph.streamNodeProperty" in query:
             graph_name = params["graph_name"]
             property_name = params["properties"]
-            return self._run_arrow_property_get(graph_name, "NODE", property_name)
+            return self._run_arrow_property_get(
+                graph_name, "gds.graph.streamNodeProperty", {"nodeProperty": property_name}
```

Comment:
We changed this to `node_property` and `node_labels`.

Reply:
Looks like it's `nodeProperty` and `nodeLabels`. Updated it to that and works now.